jdaviz/core/helpers.py

```diff
@@ -315,6 +315,32 @@ def show(self, loc="inline", title=None):
         as only JupyterLab has a mechanism to have multiple tabs.
         """
         try:
+            # Check if the user is running Jdaviz in the correct environments.
+            # If not, provide a friendly msg to guide them!
+            try:
+                shell = get_ipython().__class__.__name__  # noqa
+                if shell == 'TerminalInteractiveShell':
+                    raise RuntimeError("IPython shell not supported")
+                elif shell != 'ZMQInteractiveShell':
+                    raise RuntimeError("Unknown shell not supported")
+            except Exception as orig_e:
+                if type(orig_e) is NameError:
+                    orig_e = RuntimeError("Standard Python Interpreter not supported")
+
+                boilerplate_e = \
+                    RuntimeError('\nYou are currently running Jdaviz from an unsupported '
+                                 'shell. Jdaviz is intended to be run within a Jupyter '
+                                 'notebook, or directly from the command line.\n\nTo run '
+                                 'from Jupyter, call <your viz>.show() from a notebook cell.\n'
+                                 'To run from the command line, run '
+                                 'jdaviz --layout=<your viz> outside of Python.\n\n'
```

Comment:
I think this is out-of-date.  Perhaps just suggest `jdaviz --help`?

Reply:
Whoops, force of habit! Should I correct this or suggest `--help`? I wanted to give the user an answer for their problems, but that puts it on us to keep this message up to date; calling `--help` is probably more sustainable, but puts another step upon the user. What do you think?